app/src/list-data/conversion-types.ts

```diff
@@ -2,72 +2,165 @@ import { EntityTypes, NrRequestTypeCodes } from '@/enums'
 import { ConversionTypesI } from '@/interfaces/models'
 
 export const ConversionTypes: ConversionTypesI[] = [
+  // Regular Businesses
```

Comment:
Super small but if you change it to this then it will look like a "block comment":
```
//
// Regular businesses
//
```